Core/src/Geometry/CuboidVolumeBuilder.cpp

```diff
@@ -98,6 +98,15 @@ std::pair<double, double> Acts::CuboidVolumeBuilder::binningRange(
   // Construct return value
   std::pair<double, double> minMax = std::make_pair(
       std::numeric_limits<double>::max(), -std::numeric_limits<double>::max());
+
+  //Compute the min volume boundaries for computing the binning start
+  //See https://acts.readthedocs.io/en/latest/core/geometry.html#geometry-building
+  // !! IMPORTANT !! The volume is assumed to be already rotated into the telescope geometry 
+  Vector3 min_volume_boundaries = cfg.position - 0.5*cfg.length;
+  Vector3 max_volume_boundaries = cfg.position + 0.5*cfg.length;
```

Comment:
Minor comment would be that we tend to use camelCase, so `minVolumeBoundaries` etc.

Reply:
Done